coderedcms/templatetags/coderedcms_tags.py

```diff
@@ -40,6 +41,26 @@ def coderedcms_version():
 def generate_random_id():
     return ''.join(random.choice(string.ascii_letters + string.digits) for n in range(20))
 
+@register.simple_tag(takes_context=True)
+def twitter_card_image(context, page):
+    if page.cover_image:
+        print("cover image")
+        return page.cover_image.get_rendition('original').url
+    elif LayoutSettings.for_site(context['request'].site).logo:
+        layout_settings = LayoutSettings.for_site(context['request'].site)
+        print("layout settings logo")
+        return layout_settings.logo.get_rendition('original').url
+
+@register.simple_tag(takes_context=True)
+def og_image(context, page):
+    if page.og_image:
+        return page.og_image.get_rendition('original').url
+    elif page.cover_image:
+        return page.cover_image.get_rendition('original').url
+    elif LayoutSettings.for_site(context['request'].site).logo:
+        layout_settings = LayoutSettings.for_site(context['request'].site)
+        return layout_settings.logo.get_rendition('original').url
```

Comment:
It would be nice if this function returned the full URL (`http://example.com/image.jpg`) rather than just the image URL (`/image.jpg`)